src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+
+function Pagination({ pageContext, type }) {
+  let { currentPage, totalPages, minPage, maxPage, path } = pageContext
+
+  const apiPageChange = value => {
+    if (value !== currentPage) pageContext.formData(pageContext.formResponse, value)
+  }
+
+  const direction = value => {
+    let count = value.target.innerText === 'More' ? 10 : -10
+    pageContext.setPageContext({
+      currentPage: currentPage,
+      totalPages: totalPages,
+      minPage: minPage + count,
+      maxPage:
+        count === 'More'
+          ? maxPage + 10 > totalPages
+            ? totalPages
+            : maxPage + 10
+          : maxPage + count,
+      setPageContext: pageContext.setPageContext,
+      formResponse: pageContext.formResponse,
+      formData: pageContext.formData,
+    })
+  }
+
+  if (type === 'route') {
+    return (
+      <>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-around',
+            alignItems: 'center',
+            margin: '0 auto',
+          }}
+        >
+          <nav aria-label="Page navigation">
+            <ul className="pagination">
+              {currentPage > 1 && (
+                <li>
+                  <Link to={`/${path}/${currentPage === 2 ? '' : currentPage - 1}`} rel="prev">
+                    {/* <span aria-hidden="true">&laquo;</span> */}
+                    <span aria-hidden="true">Previous</span>
+                  </Link>
+                </li>
+              )}
+              {Array.from({ length: totalPages }, (_, index) => (
+                <li key={index}>
```

Comment:
Index as a key is an anti-pattern
What happens if you push an item to the list or remove something in the middle? If the key is same as before React assumes that the DOM element represents the same component as before. But that is no longer true.

Reply:
The key is used to display the page number and cant be edited. 
![image](https://user-images.githubusercontent.com/15128241/131913035-94c49085-a5b2-463c-ade7-50b6a7fd94da.png)

Reply:
The `key` is used for React to track changes in the list. You can still use `index` in L54. the key has to be something different

Reply:
Thanks Steven, I should have looked closer. I see it now. I'm killing the key.

Joe

Reply:
Done